src/components/table/table-meta-header/TableMetaHeader.tsx

```
@@ -63,26 +63,24 @@ export default function TableMetaHeader({
+      }
+      return {
+        key: filter.key,
+        values: defaultValues,
+        type: filter.type,
       }
-
-      setDefaultFilters([...filters?.map(mapFn)])
     }
-  }, [filters?.length, filterConfig])
+
+    setDefaultFilters([...filters?.map(mapFn)])
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)